flake.nix

```diff
@@ -0,0 +1,52 @@
+{
+  description = "Rustlings devShell.";
```

Comment:
Might be better to use the description in this repo.

```suggestion
  description = "Small exercises to get you used to reading and writing Rust code";
```

Reply:
:+1: